src/csharp/Microsoft.Spark/Sql/DataFrame.cs

```diff
@@ -535,6 +535,14 @@ public DataFrame Hint(string name, object[] parameters = null)
         public DataFrame Observe(string name, Column expr, params Column[] exprs) =>
             WrapAsDataFrame(_jvmObject.Invoke("observe", name, expr, exprs));
 
+        /// <summary>
+        /// Create a write configuration builder for v2 sources.
+        /// </summary>
+        /// <param name="table">Name of table to write to</param>
+        /// <returns>DataFrameWriterV2 object</returns>
+        public DataFrameWriterV2 WriteTo(string table) =>
```

Comment:
Can we add `[Since(Versions.V3_0_0)]` to this function?

Reply:
Added. Thanks!